### PR TITLE
chore: Remove the unused AuthHelper

### PR DESCRIPTION
**Pull request by royjit**
*Description of changes:*

*Check points:*

- [ ] ~Added new tests to cover change, if needed~
- [x] All unit tests pass
- [ ] ~All integration tests pass~
- [ ] ~Documentation update for the change if required~
- [x] PR title conforms to conventional commit style

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by diegocstn — APPROVED** [review]

File: Podfile.lock [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+  - AWSCognitoIdentityProvider (2.24.4):
+    - AWSCognitoIdentityProviderASF (= 2.24.4)
+    - AWSCore (= 2.24.4)
+  - AWSCognitoIdentityProviderASF (2.24.4)
+  - AWSCore (2.24.4)
+  - AWSMobileClient (2.24.4):
+    - AWSAuthCore (= 2.24.4)
+    - AWSCognitoIdentityProvider (= 2.24.4)
+    - AWSCognitoIdentityProviderASF (= 2.24.4)
+    - AWSCore (= 2.24.4)
```
Comment (lawmicha) [inline Podfile.lock:13]: do we need to update the rest of the lock files?

File: AmplifyTestCommon/Helpers/AuthHelper.swift [hunk 2]
```diff
@@ -1,107 +0,0 @@
...
-// Copyright Amazon.com Inc. or its affiliates.
-// All Rights Reserved.
-//
-// SPDX-License-Identifier: Apache-2.0
-//
-
-import Foundation
-import AWSMobileClient
-
-class AuthHelper {
```
Comment (lawmicha) [inline AmplifyTestCommon/Helpers/AuthHelper.swift:11]: are all the plugin tests off of this AuthHelper?

File: Podfile.lock (hunk 1 above)
Comment (royjit) [reply Podfile.lock:13]: Added

File: AmplifyTestCommon/Helpers/AuthHelper.swift (hunk 2 above)
Comment (royjit) [reply AmplifyTestCommon/Helpers/AuthHelper.swift:11]: Yes, couldnot find any reference to this one.

**Review by lawmicha — APPROVED** [review]

**Review by diegocstn — APPROVED** [review]

**Pull request merged**